guides/quickstart.md

```diff
@@ -0,0 +1,175 @@
+
+### Configuring a Source type
+
+In taint propagation lingo, a value that contains data of interest is called a Source.
+In the current example, a Source is a value that contains sensitive data, e.g. a password.  
+
+There are 2 ways to define a Source type:
+1. Define a field tag and apply it to sensitive fields
+2. Describe a type using its name, the name of its package, and the relevant field(s)
+
+#### Using field tags
+
+In the analyzer's configuration, define the field tag:
+
+`analyzer_configuration.yaml`
+```yaml
+FieldTags:
+  - Key: datapolicy
+    Val: secret
+```
+
+In the code you want to analyze, add the tag to the sensitive field:
+
+`auth.go`
+```go
+type Authentication struct {
+  Username string 
+  Password string `datapolicy:"secret"`
+}
+```
+
+If you do not wish to define your own field tag, you may use the built-in `levee:"source"` tag.
+
+#### Using type descriptions
+
+In the analyzer's configuration, identify the sensitive data:
+
+`analyzer_configuration.yaml`
+```yaml
+Sources:
+  - Package: auth
+    Type: Authentication
+    Field: Password
+```
+
+This method does not require changes to the code.
+
+See [the documentation](../configuration/README.md) for further instructions on how to describe sources.
+
+### Configuring a Sink
+
+In taint propagation lingo, a function that should not be called with a tainted value is called a "Sink".
+In the current example, a Sink is a function that could leak a sensitive value, e.g. by writing it to a log.
+
+Configuring a sink is similar to configuring a source using a type description.
+In the analyzer's configuration, identify the function:
+
+```yaml
+Sinks:
+  - Package: log
+    Method: Printf
+```
+
+See [the documentation](../configuration/README.md) for further instructions on how to describe sinks.
+
+## Running the analyzer
+
+Now that the analyzer knows what our Sources and Sinks are, we can run it to detect the issue in the example code.
+
+### Install the analyzer
+
+Use the following command to install the analyzer:
+
+```shell
+go get github.com/google/go-flow-levee/cmd/levee
+```
+
+### Run the analyzer
+
+The most convenient way to run the analyzer is to use the `go vet` command.
+
+You must provide `go vet` with three pieces of information:
+1. What tool to run (in this case, the levee analyzer)
+2. What configuration the analyzer should use
+3. The list of packages to analyze
+
+Use the following command to run the analyzer:
+```shell
+go vet -vettool=$(which levee) -config=$(realpath analyzer_configuration.yaml) ./...
+```
+
+(This command assumes that it is being run in the `go-flow-levee/guides/quickstart` directory.)
+
+Running on the example code, you should see output similar to the following:
+
+```
+# github.com/google/go-flow-levee/guides/quickstart
+./auth.go:13:13: a source has reached a sink
+ source: ./auth.go:10:19
+```
```

Comment:
I don't think there's anything for it, but this documentation is going to go stale pretty quickly.  Something to keep in mind down the road.

Reply:
I hope it does :sweat_smile:
We'll update it when it does